extensions/http/src/test/resources/basic-http-config.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:httpn="http://www.mulesoft.org/schema/mule/httpn"
+      xmlns:spring="http://www.springframework.org/schema/beans"
+      xmlns="http://www.mulesoft.org/schema/mule/core"
+      xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+      http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-current.xsd
+      http://www.mulesoft.org/schema/mule/httpn http://www.mulesoft.org/schema/mule/httpn/current/mule-httpn.xsd">
+    
+    <httpn:request-config name="reqConfig" host="localhost" protocol="HTTP" port="${clientPort}">
+        <httpn:requester-connection/>
+    </httpn:request-config>
+    
+    <httpn:listener-config name="lisConfig">
+        <httpn:listener-connection host="localhost" port="${serverPort}" protocol="HTTP"/>
+    </httpn:listener-config> 
+    
+    <flow name="client">
+        <httpn:request path="/" config-ref="reqConfig" key="ANY">
+            <httpn:request-builder>
+                <httpn:requester-request-builder>
```

Comment:
as you think it's best, but `requester-request-builder` definitively doesn't sound good